debug/timedim.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
```

Comment:
In this example I'm currently getting this error: "CartoParsingError: Invalid function name 'clusterTimeStart'."